src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx

```diff
@@ -58,6 +58,10 @@ const parseRange = (range?: string) => {
   })
 }
 
+const getValue = (value: CustomRange[number]) => {
+  return value === "*" || value === 0 ? "" : value
```

Comment:
I always forget this but clearing a value for an input in React: use empty string instead of `null`